ingress/controllers/nginx/nginx.tmpl

```diff
@@ -156,6 +156,12 @@ http {
     }
     {{ end }}
 
+    {{/* build all the required rate limit zones. Each annotation requires a dedicated zone */}}
+    {{/* 1MB -> 16 thousand 64-byte states or about 8 thousand 128-byte states */}}
+    {{ $zone := range (buildRateLimitZones .servers) }}
```

Comment:
I'm late to the battle but shouldn't this be `{{ range $zone := (buildRateLimitZones .servers) }}`?

Should I make a PR?